bin/node/src/aleph_node_rpc.rs

```diff
@@ -125,4 +178,31 @@ where
             })?;
         Ok(())
     }
+
+    fn aleph_node_block_author(&self, hash: Block::Hash) -> RpcResult<Option<AccountId>> {
+        let header = self.client.header(hash).unwrap().unwrap();
+        if header.number().is_zero() {
+            return Ok(None);
+        }
+
+        let slot = header
+            .digest()
+            .logs()
+            .iter()
+            .find_map(<DigestItem as CompatibleDigestItem<Signature>>::as_aura_pre_digest)
+            .ok_or(Error::BlockWithoutDigest)?;
+
+        let parent = header.parent_hash();
+
+        let block_producers_at_parent = self
+            .client
+            .runtime_api()
+            .session_validators(*parent)
```

Comment:
This is problematic because you just introduced this API, so it will not work for old runtimes...

Reply:
Is it possible to use runtime_api().authorities()?
1. Are those equivalent to `session.validators()`
2. Is it possible to convert `TUhorityId` to `Account`?

Reply:
nope, the main problem is that `authorities()` wasn't present from the very beginning and we want this endpoint to wrok for all runtime versions :/ also, `authorities()` would return aleph session keys, which would have to be mapped to accounts again, with runtime, so not really helpful

Reply:
I'm working on a workaround with raw storage reading

Reply:
fixed - I'm reading `Session::Validators` storage item now